src/utils/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface ClassNameProps {
```

Comment:
`/utils` folder = some helpers, types are more suitable to be located in in `/types` folder, but is it nesessary to create interface for one className prop? Are there any potential others props for ClassNameProps?
May be better to use simple className prop where needed same with `uikit` approach

Reply:
I find it pretty convenient to use with `ClassNameProps` helper approach, but we can drop it in favour of `uikit` approach